### PR TITLE
Add back generate command

### PR DESCRIPTION
**Pull request by dillonraphael**
Closes: #3691 

### What are the changes and their implications?

- Add back the generate command
- Clean up help message
- Fix `determineContext` and only run if no context is supplied & the `--context` flag is explicitly used
- Run `blitz codegen` after generating pages

_Note: This PR also fixes an issue with generating a new blitz app_

### Todo

- [X] Use `Routes` object in generator templates
- [x] Update generator templates

### CODE REVIEW

**Comment by dillonraphael** [comment]
@beerose Do you think we should update the generator templates for mutations & queries to use resolvers?

**Comment by beerose** [comment]
> @beerose Do you think we should update the generator templates for mutations & queries to use resolvers?

Yes, we should.

File: packages/generator/src/generator.ts [hunk 1]
```diff
@@ -302,7 +302,7 @@ export abstract class Generator<
       return path.join(
         __dirname,
         "..",
-        process.env.NODE_ENV === "test" ? "./templates" : "./dist/templates",
+        process.env.NODE_ENV === "test" ? "./templates" : "./templates",
```
Comment (beerose) [inline packages/generator/src/generator.ts:305]: I think it needs to be `dist/templates` when you're using it outside of monorepo.

Comment (dillonraphael) [reply packages/generator/src/generator.ts:305]: Oops ya, this is a mistake. Thanks for finding.

**Pull request merged**